lua/telescope/builtin/internal.lua

```diff
@@ -81,7 +81,23 @@ internal.builtin = function(opts)
     previewer = previewers.builtin.new(opts),
     sorter = conf.generic_sorter(opts),
     attach_mappings = function(_)
-      actions.select_default:replace(actions.run_builtin)
+      actions.select_default:replace(function(_)
+        local selection = action_state.get_selected_entry()
+        if not selection then
+          print "[telescope] Nothing currently selected"
+          return
+        end
+        if string.match(selection.text, " : ") then
+          -- Call appropriate function from extensions
+          local split_string = vim.split(selection.text, " : ")
+          local ext = split_string[1]
+          local func = split_string[2]
+          require("telescope").extensions[ext][func]()
+        else
+          -- Call appropriate telescope builtin
+          require("telescope.builtin")[selection.text]()
```

Comment:
We still need to pass in the opts so the theme works.
And it probably makes sense to nil `ignore_filename` and `include_extensions` the two opts that builtin supports